sinopia_client/test/api/LDPApi.spec.js

```diff
@@ -241,18 +263,36 @@
               expect(JSON.parse(responseAndData.response.text)).to.deep.equal(JSON.parse(nonRdfResourceJson))
             })
         })
-
-        describe.skip('on an RDF resource', function() {
-          // TODO: a test: https://github.com/LD4P/sinopia_server/issues/34
-        })
       })
+
       describe('on an RDF resource', function() {
+        let rdfResourceJson = fs.readFileSync(path.join(__dirname, '../../../fixtures/resource_defs/resource1.jsonld'), { encoding: 'utf8' })
+        let groupId = 'rdfStuff'
+        let slug = 'rdfResource'
+
         // need a group container to exist to hold our resource
         beforeEach(function() {
           let grpRsrcCtx = new SinopiaServer.SinopiaBasicContainerContext('http://www.w3.org/2000/01/rdf-schema#', 'http://www.w3.org/ns/ldp#')
-          let groupRsrc = new SinopiaServer.SinopiaBasicContainer('', grpRsrcCtx, ['ldp:Container', 'ldp:BasicContainer'], 'Profiles Group')
+          let groupRsrc = new SinopiaServer.SinopiaBasicContainer('', grpRsrcCtx, ['ldp:Container', 'ldp:BasicContainer'], 'Retrievable Resources Group')
           return instance.createGroup('rdfStuff', groupRsrc)
         });
+        // need an rdf resource to retrieve
+        beforeEach(function() {
+          let opts = {
+            'slug': slug,
+            'contentType': 'application/ld+json',
+            'link': '<http://www.w3.org/ns/ldp#RDFSource>; rel="type"'   //TODO: centralize type strings?  https://github.com/LD4P/sinopia_server/issues/68
+          }
+          return instance.createResource(groupId, rdfResourceJson, opts)
+        })
+
+        it('should return the expected content', function() {
+          return instance.getResource(groupId, slug, { accept: 'application/ld+json' })
```

Comment:
could've also done `getResourceWithHttpInfo`, and looked at the headers ot make sure `Link: <http://www.w3.org/ns/ldp#RDFSource>; rel="type"` was there.  would be open to doing that instead, or as an additional test.  🤷‍♂

Reply:
:man_shrugging: indeed